api/client/options.proto

```diff
@@ -56,6 +56,9 @@ message CommandLineOptions {
   google.protobuf.UInt32Value max_active_requests = 15 [(validate.rules).uint32 = {gte: 1}];
   // See :option:`--max-requests-per-connection` for details.
   google.protobuf.UInt32Value max_requests_per_connection = 16 [(validate.rules).uint32 = {gte: 1}];
+  // See :option:`--output-format` for details.
+  google.protobuf.StringValue sequencer_idle_strategy = 17
```

Comment:
I think this might be better as an enum.

Reply:
@mergeconflict 
So right now this is consistent with the other candidates that would be better represented as an enum.
I agree that an `enum` is the better option, so what I would propose here is leaving this as-is in this PR, and subsequently update all candidates in one go, in a single followup.

Reply:
Yeah, I think a followup `enum` PR (before 0.2 release) makes sense.

Reply:
In progress over at https://github.com/oschaaf/nighthawk/tree/proto-enums